src/meteors/visualize/shap/plots.py

```diff
@@ -0,0 +1,691 @@
+from __future__ import annotations
+
+from matplotlib.axes import Axes
+from matplotlib.colors import Colormap
+from matplotlib.figure import Figure
+
+from meteors.shap import HyperSHAP, SHAPExplanation
+
+import shap
+import numpy as np
+import pandas as pd
+
+import matplotlib.pyplot as plt
+
+
```

Comment:
Move shap visualizations alongside the attr and hsi _visualizations making `shap_visualizations`

Reply:
fixed in [913f8e7](https://github.com/xai4space/meteors/pull/102/commits/913f8e75ed91715415d0924c4aa060038ea8a1a1)